### ISSUE TITLE
cult comms 2hard

### ISSUE THREAD

**Issue by Allspice** [vgstation-coders/vgstation13#18093]
tl;dr: one-click comms from tome to actually get cultist activities going and to prevent roundstart scrubs exposing the cult when
---
Cult 3 soonish eventually sure, but I think it's still worth bringing up. Starting first with some minor analysis:

Team antags with communication superior to cultists': Xeno; Malf; Ops; Raider (hue)
Potential team antags with superior communication: Blob
Non-team antags with superior communication: Ling
Potential team antags with inferior communication: Wiz; Vamp; Rev.

Examining that final category for a moment: Wiz and rev are both very mechanically simpler than cult, and in practice need the communication less because (unless the wiz is doing gimix) they both heavily incentivize shock and awe to dominate the station and/or get that greentext. And vamp is sort of complicated to newbies, but once a vamp can start thralling it's generally swole enough to fuck people up even without thralls.

On the other hand, cult has built-in roadblocks slowing it down (research and talismans and optional base-building in cult 2, the starting altar and act completion and slower conversion and somewhat more obligate base-building and talismans in cult 3). So there's a much greater communication requirement than "flash people and welderbomb CMO". But as things stand, cult comms are bad and feel like trash to use in a nontrivial fashion. Either you need to bug off to a secluded place and maek rune and say all the shit that you want said then go back to your job (and repeat the process if someone else issues a reply that you need to make a response to). Or you and every other person that you'd hope to communicate with in a significant capacity, needs to keep a stock of comm talismans on your person, and do an unnatural word-vomit of every relevant detail that you're telling the other cultists to avoid using the talismans too quickly (like you'd do if you used a comm talisman per sentence). Both measures feel pretty janky for the most communication-dependent antag. And all of that makes it feel like trash to coordinate real swanky culty shit because it feels like trash just to god damn talk to cultists.

And finally: Since cult isn't whitelisted, the group of roundstart cultist can and often does include new people. I'm not advocating for whitelisting, but it's a simple fact that new cultists that do cultist things are more likely to expose the cult. The chance of this happening are increased, particularly near roundstart, because it's hard to coordinate between cultists with limited-charge supply talismans, either to ask for help or to tell the newbies what pitfalls to avoid. And I don't know the numbers off the top of my head, but it sure feels like 25-30% of cults are exposed because one of the starters left some stupid shit in maint or whatever 10 minutes in, so it's a big problem.

**Comment by SonixApache** [comment]
just do what TEEGEE does

**Comment by Pathid** [comment]
On TEEGEE you only need your hands free, right? Not even a tome? We don't necessarily need to go that far. Could make communication runes/talismans multi-use instead.

**Comment by Aurx** [comment]
Why not make the communication rune/talisman enable a cultspeak verb for some period of time?

**Comment by Waismuth** [comment]
TG's cult seemed far easier by comparison, you had the option to telepathically talk with other cultist via the cult chat at any given moment, and you didn't need to research words. The option for all-time cult chat would be really good, even simple requests like "please don't put runes and leave em unattended; keep low while I research." oh and another thing, there should be a strong OOC note when you roundstart a cult making it clear and crystal you HAVE TO  share the cult word, like, to make it clear you can be a helped if you dont. If you don't want to play a cult don't check it in antag prefs, but Ive seen a lot of times that people simply dont share their words, and there should be a way of making people are of that they need to share it

**Comment by Aurx** [comment]
Other idea.  Cult3 is about base building?  Make a cultchat cell tower.

**Comment by Blithering** [comment]
Converting certain runes to buffs would be neat, another one that could use this is see invisible. Currently you need to stand on and channel it to see hidden runes, making hiding runes in the first place kinda pointless as there's gonna be one visible in any case

**Comment by Pathid** [comment]
@Blithering There's a way to do it so it's hidden. A couple ways.

**Comment by Blithering** [comment]
@Pathid hidden, or invisible?

**Comment by Pathid** [comment]
@Blithering Invisible.

**Comment by DeityLink** [comment]
...Fixed?
